hooks/useMaskParam.ts

```diff
@@ -0,0 +1,19 @@
+import { useSearchParams } from 'next/navigation';
+import { useMemo } from 'react';
+
+export function useMaskParam(): boolean {
+    const searchParams = useSearchParams();
+    const mask = searchParams.get('mask');
```

Comment:
Why `mask` though?

<img src="https://media3.giphy.com/media/vL7uCEylnzjHUHdMaP/giphy.gif"/>

Reply:
Hm... Ok. 

Re: rendering mask. Wouldn't it be better to render mask from the outside of the iframe?

Reply:
I think I've tried that but the problem was that it was not possible to scroll the iframe IIRC.